cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutorState.java

```diff
@@ -16,20 +16,27 @@
     NO_TASK_IN_PROGRESS,
     EXECUTION_STARTED,
     REPLICA_MOVEMENT_TASK_IN_PROGRESS,
-    LEADER_MOVEMENT_TASK_IN_PROGRESS;
+    LEADER_MOVEMENT_TASK_IN_PROGRESS,
+    STOPPING
```

Comment:
Should we make it `EXECUTION_STOPPING` to make it consistent with `EXECUTION_STARTED`?